RecoLocalTracker/SiPixelRecHits/plugins/alpaka/PixelRecHitKernels.dev.cc

```diff
@@ -91,40 +68,6 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE {
 #ifdef GPU_DEBUG
         alpaka::wait(queue);
 #endif
-
```

Comment:
Why this part is no longer needed ?

Reply:
Because the I've moved the part were we fill the hit histogram (that depends on how the layers are defined) in the CA chain in [`CAHitNtupletGeneratorKernels<TrackerTraits>::prepareHits`](https://github.com/cms-sw/cmssw/blob/17f86b8f731bd6adc6bd36bb1b5442a6f95abaf1/RecoTracker/PixelSeeding/plugins/alpaka/CAHitNtupletGeneratorKernels.dev.cc#L173). This is done because now the layer structure is defined in the CA itself with the `CAGeometry` and since this is only really needed by the CA (it's not used anyway nor useful for the hits).